main.cpp

```diff
@@ -13,6 +14,17 @@ class Student
             : _name(name), _surname(surname), _index(index) {}
 };
 
+class Database
+{
+    public:
+    void addstudent(std::string name, std::string surname, int index);
+    void delstudent(std::string name, std::string surname, int index);
+
+    private:
```

Comment:
@damianSt89 
- please put private section first (as you done in StudentClass)
- try to keep one style: add tab after private members and public members